src/style.css

```diff
@@ -0,0 +1,77 @@
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  font-family: Arial, sans-serif;
+  color: rgb(255, 255, 255);
+  font-size: 100px;
+
+  width: 100%;
```

Comment:
width is alright. but let's practice and discover flex and how to work with it. let's use `flex-basis`